ArduPlane/Parameters.cpp

```diff
@@ -205,6 +205,24 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(takeoff_pitch_limit_reduction_sec, "TKOFF_PLIM_SEC",  2),
 
+    // @Param: TKOFF_RDDR_FF
+    // @DisplayName: Takeoff aileron to Rudder feedforward
+    // @Description: This parameter sets aileron to rudder ff below the TKOFF_RDDR_FF_H hieght, helpful to resolve tendency of plane to roll during takeoff.
+    // @Units: fraction
+    // @Range: 0 1
+    // @Increment: 0.1
+    // @User: User
+    GSCALAR(takeoff_rudder_mix, "TKOFF_RDDR_MIX",  0),
+
+    // @Param: TKOFF_RDDR_FF_H
+    // @DisplayName: Takeoff rudder feedforward min altitude
+    // @Description: Height below which the TKOFF_RDDR_FF will be active setting it to 0 will disable the takeoff roll to rudder feedforward
+    // @Units: percent
```

Comment:
You have to use one of the units in the left column in https://github.com/ArduPilot/ardupilot/blob/master/Tools/autotest/param_metadata/param.py#L41-L96 so in this case you need to write %
I'm confused though as the description says this is a height.